### DISCUSSION TITLE
Is it possible to get allure reporter test case data in runtime

### DISCUSSION THREAD

**Discussion by Wojciech208** [webdriverio/webdriverio#12629] [Q&A]
I use allure as a webdriverIO reporter. Is it possible to get access to allure reporter data as current test case steps, and other parameters during runtime? I.e. in after suite or after hook.

**Comment by christian-bromann** [comment]
You can get WebdriverIO test case data during runtime but writing a custom [service](https://webdriver.io/docs/customservices) or [reporter](https://webdriver.io/docs/customreporter). However this is unrelated to the Allure reporter. Can you elaborate for what purpose you may need this data?
